Genome-Annotation/tutorials/general-introduction.md

```diff
@@ -1,110 +1,154 @@
+{: .question}
 
 ![augustus_output](../images/augustus_out.png)
 
+## tRNA and tmRNA Prediction
 
-## 2.3 tRNA and tmRNA Prediction
+Use **Aragorn** for tRNA and tmRNA prediction. As input file use the *Aspergillus* genome sequence. You can choose the genetic code (e.g. bacteria). Select the topology of your genome (circular or linear).  Are there tRNAs or tmRNAs in the sequence?
 
-Use **Aragorn** for tRNA and tmRNA prediction. As input file use the *Aspergillus* genome sequence. You can choose the genetic code (e.g. bacteria). Select the topology of your genome (circular or linear). Read more about **Aragorn** [here](http://nar.oxfordjournals.org/content/32/1/11.full.pdf+html). Are there tRNAs or tmRNAs in the sequence?
+> ### :bulb: Tip:
+>
+> read more about **Aragorn** [here](http://nar.oxfordjournals.org/content/32/1/11.full.pdf+html).
+{: .tip}  
 
+# Functional Annotation
 
-# 3. Functional Annotation
-
-## 3.1 Similarity Searches (BLAST)
+## Similarity Searches (BLAST)
 
 Functional gene annotation means the description of the biochemical and biological function of proteins. Possible analyses to annotate genes can be for example:
 
-
 - similarity searches
 - gene cluster prediction for secondary metabolites
 - identification of transmembrane domains in protein sequences
 - finding gene ontology terms
 - pathway information
 
+For similarity searches we use *NCBI BLAST+ blastp* to find similar proteins in a protein database.
 
-For similarity searches we use *NCBI BLAST+ blastp* to find similar proteins in a protein database. As input file, select the protein sequences from Augustus, choose the protein BLAST database *SwissProt* and the output format *xml*. 
+### Similarity search
 
+:wrench: As input file, select the protein sequences from Augustus, choose the protein BLAST database *SwissProt* and the output format *xml*. 
 
 <img src="../images/blastP.png" width="70%">
 
+Parsing the xml output (**Parse blast XML output**) results in changing the format style into tabular.
 
+> ### :question: Questions
+>
+> What information do you see in the BLAST output?
+>
+> <details>
+> <summary></summary>
+> </details>
+{: .question}
 
-Parsing the xml output (**Parse blast XML output**) results in changing the format style into tabular. What information do you see in the BLAST output?
-
-From BLAST search results we we want to get only the best hit for each protein. Therefore apply the tool **BLAST top hit descriptions** with *number of descriptions =1* on the xml output file. For how many proteins we do not get a BLAST hit? Choose the tool **Select lines that match an expression** and enter the following information: *Select lines from* [select the BLAST top hit descriptions result file]; *that* [not matching]; *the pattern* [gi]. 
+From BLAST search results we want to get only the best hit for each protein.
+:wrench: Therefore apply the tool **BLAST top hit descriptions** with *number of descriptions =1* on the xml output file. 
```

Comment:
Hands-on box? Maybe combining all the previous ones in this section in a big hands-on box with question boxes inside. What do you think?